runtime/src/iree/builtins/ukernel/mmt4d_internal.h

```diff
@@ -53,15 +53,17 @@ static inline iree_uk_type_t iree_uk_mmt4d_out_type(iree_uk_mmt4d_type_t type) {
 // good, because it's hard to get the struct field offsets right in assembly
 // and keep that in sync with future struct changes.
 typedef void (*iree_uk_mmt4d_tile_func_t)(
-    void* /*out_tile*/, const void* /*lhs_panel*/, const void* /*rhs_panel*/,
-    iree_uk_int32_t /*K*/, iree_uk_uint32_t /*flags*/,
-    const iree_uk_mmt4d_params_t* /*params*/);
+    void* IREE_UK_RESTRICT /*out_tile*/,
```

Comment:
note that you can put the arg names in typedefs and that helps IDE's/completion that uses them
![image](https://user-images.githubusercontent.com/75337/235971275-381286b8-adac-46a2-8c4b-f96a76ab3d1e.png)

Reply:
Done!